templates/analytics/support.html

```diff
@@ -18,11 +18,21 @@
         </center>
     </form>
 
-    {% if message %}
+
+    {% if error_msg %}
+    <div class="alert alert-danger">
+        <center>
+            {{ error_msg }}
+        </center>
+    </div>
+    {% elif message %}
     <div class="alert alert-success">
         <center>
             {{ message }}
```

Comment:
We should probably rename `message` to `success_message`

Reply:
renamed every other instance of `context["message"]` to `context["success_message"]` and have made the corresponding change here as well...